main/modes/games/danceNetwork/danceNetwork.c

```diff
@@ -127,7 +127,7 @@ const trophySettings_t danceNetworkTrophySettings = {
     .drawFromBottom   = false,
     .staticDurationUs = DRAW_STATIC_US * 6,
     .slideDurationUs  = DRAW_SLIDE_US,
-    .namespaceKey     = danceNetworkName,
```

Comment:
@johnnywycliffe Turns out this was too long, and it immediately crashed and I noticed it and fixed it. Thanks past me!